projects/novo-elements/src/elements/form/extras/address/Address.ts

```diff
@@ -2,7 +2,7 @@
 import { Component, forwardRef, Input, OnInit, ChangeDetectionStrategy, EventEmitter, Output } from '@angular/core';
 import { NG_VALUE_ACCESSOR, ControlValueAccessor } from '@angular/forms';
 // APP
-import { getCountries, getStates, findByCountryId } from '../../../../utils/countries/Countries';
+import { getCountries, getStates, findByCountryId, COUNTRIES } from '../../../../utils/countries/Countries';
```

Comment:
is getCountries() still used at all here?

Reply:
Fixed